pkg/tests/cross-tests/input_check.go

```diff
@@ -113,11 +119,24 @@ func runCreateInputCheck(t T, tc inputTestCase) {
 
 	pt.Up()
 
-	// TODO: verify that these comparisons ensure full equality.
-	// compare the two inputs
-	assertCtyValEqual(t, "RawConfig", tfResData.GetRawConfig(), pulResData.GetRawConfig())
-	assertCtyValEqual(t, "RawPlan", tfResData.GetRawPlan(), pulResData.GetRawPlan())
+	for k := range tc.Resource.Schema {
+		// TODO: make this recursive
```

Comment:
Is there some existing thing for iterating recursively over a TF schema?

Reply:
Search for "walk" in this repo for something that can visit shim.Schema; for sdkv2.Schema may be worth searching their lib but I don't remember off-hand.